modules/tensor_mechanics/include/materials/ADComputeIncrementalShellStrain.h

```diff
@@ -160,6 +160,7 @@ class ADComputeIncrementalShellStrain : public Material
 
   /// Rotation matrix material property
   std::vector<MaterialProperty<RankTwoTensor> *> _rotation_matrix;
+  std::vector<const MaterialProperty<RankTwoTensor> *> _rotation_matrix_old;
```

Comment:
This property is added but never used. Why?

Reply:
Had to add this property because _rotation_matrix is calculated at the beginning of the simulation only. So, when the stresses/strain are being transformed the _rotation_matrix of the last element was used. This would create no any problem for uniform mesh but for nonuniform mesh like tapered section the _rotation_matrix for the elements are different due to which I had to store the _rotation_matrix for all elements.